frontend/src/components/Pages/MissionPage/TaskOverview/TaskTable.tsx

```diff
@@ -7,8 +7,12 @@ import { tokens } from '@equinor/eds-tokens'
 import { getColorsFromTaskStatus } from 'utils/MarkerStyles'
 
 const StyledTable = styled(Table)`
-    grid-column: 1/ -1;
-    font: equinor;
+    display: grid;
```

Comment:
Content of table doesn't align with table header
![image](https://github.com/equinor/flotilla/assets/33518988/27045c8a-2edf-4956-bedf-90c98da8f965)